src/bots/telegram/client.js

```diff
@@ -222,6 +226,17 @@ function prepareMessage(msg: Telegram$Message): string {
   return text;
 }
 
+function prepareQuote(quote) {
```

Comment:
This function needs flow type annotations, I guess.

Reply:
done